From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Thu, 7 Aug 2025 13:57:03 +0800
Subject: meson.build: Drop pango dependency

It's not used and won't get pulled in automatically since GTK was removed.
---
 meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/meson.build b/meson.build
index 5fcf2d8..ba62d9c 100644
--- a/meson.build
+++ b/meson.build
@@ -108,7 +108,6 @@ libcanberra_dep = dependency('libcanberra')
 libgeoclue_dep = dependency('libgeoclue-2.0', version: '>= 2.3.1')
 libnotify_dep = dependency('libnotify', version: '>= 0.7.3')
 libpulse_mainloop_glib_dep = dependency('libpulse-mainloop-glib', version: '>= 2.0')
-pango_dep = dependency('pango', version: '>= 1.20.0')
 polkit_gobject_dep = dependency('polkit-gobject-1', version: '>= 0.114')
 upower_glib_dep = dependency('upower-glib', version: '>= 0.99.12')
 x11_dep = dependency('x11')
